README.md

```diff
@@ -17,17 +17,28 @@ in more powerful backends, like `Mesos`, for large scale production deployments.
 
 ## Installation
 
-###1 - Download and install the current source code.
+###1 - Docker image.
+The easiest way to get started with Swarm is to use the
+[official Docker image](https://registry.hub.docker.com/_/swarm/).
+
+```sh
+docker pull swarm
```

Comment:
We want people to upgrade swarm eventually, getting them familiar with `pull` seemed the way to go